cmd/schema-gen/extract/parser.go

```diff
@@ -25,6 +25,7 @@ type ExtractOptions struct {
 	includeTests      bool
 	includeUnexported bool
 	ignoreFiles       []string
+	IncludeInternal   bool
```

Comment:
```suggestion
	includeInternal   bool
```

follow existing style, these are not for import

Reply:
Ok, well, if they are for import, move this type to model/